site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -75,6 +82,10 @@ Mouse users will show interest in an element by mouse-hovering the element for a
 
 For keyboard users, the UA will provide a hot-key such as Alt-Down, which will "show interest" in the element. Another hot-key such as the ESC key will provide a way to trigger "loseinterest". These hot-keys must be chosen by the UA to be convenient for the user, while also not conflicting with existing UA-provided and OS-provided hot-keys.
```

Comment:
I know this isn't settling on a key yet, but we sould look at alt+space, ctrl+space as well.
Alt+down might be nice, admittedly. I think Windows already supports it for opening a `<select size=1>` What would happen if the select also had an interest target?

Reply:
Good questions. I've added alt-space and ctrl-space as other ideas down in the `### Keyboard` section.

We'd have to make sure we resolve any overlapping hotkey problems, such as the `<select>` one you mention. I'd think we shouldn't override existing hotkeys, right? So the `<select>` would win in this case.

Reply:
Makes sense to me.